.docker/docker-compose.test.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.4'
+
+services:
+  app:
+    build:
+      target: test
+    image: libero/article-store:${IMAGE_TAG:-master}-dev
```

Comment:
Not understanding, if the original point was to run the unit tests quickly (without starting the database), removing the volumes means you don't see the code changes when starting a new container; hence you need to rebuild the image every time you run the unit tests?